source/module/plot/gmt_custom_symbols_2.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+gmt begin custom_symbol png,pdf
+gmt basemap -R0/10/-2/4 -JX10c/2c -B2
+# 绘制城市符号
+echo 1 1 | gmt plot -Skcity/1 -W0.6p
```

Comment:
注释在上会显得脚本很长，如下修改？如果可以，请修改全部注释。
```suggestion
echo 1 1 | gmt plot -Skcity/1 -W0.6p  # 绘制城市符号
```

Reply:
我觉得还行吧，又不会很长